environments/production/inventory.yml

```diff
@@ -2,6 +2,31 @@
 
 all:
   vars:
+    # Prometheus vars
     prometheus_version: 2.22.0
+
+    prometheus_web_listen_address: '0.0.0.0:9090'
+
+    prometheus_scrape_jobs:
+    - job_name: 'signal-tower'
+      metrics_path: /metrics
+      params:
+        module: [http_2xx]
+      static_configs:
+        - targets:
+          - localhost:4233
+
+    # Grafana vars
+    grafana_security: 
+      admin_user: admin
+      admin_password: "admin"
```

Comment:
Even though it is a default value I would prefer seeing this value in a seperate, vaulted file

Reply:
I create an issue #15